.github/workflows/regression_test_summariseEpoch.yml

```diff
@@ -0,0 +1,43 @@
+name: Python - summariseEpoch.py
```

Comment:
Perhaps also good to change the name of this workflow into something more meaningful. See https://github.com/activityMonitoring/biobankAccelerometerAnalysis/pull/84 here for what those checks look like.